docs/DeveloperGuide.md

```diff
@@ -141,10 +141,10 @@ Commands that extends the UndoableCommand class have a concrete implementation o
 The `Model` component,
 
 * stores the address book data i.e., all `Person` objects (which are contained in a `UniquePersonList` object).
-* stores the current list of tags as an `ObservableList<Tag>`. This allows `AddessBook` to only require one `Tag` object per unique tag, instead of each `Person` needing their own `Tag` objects.
+* stores the current list of tags as an `ObservableList<Tag>` that can be 'observed' e.g. the UI can be bound to this list so that the UI automatically updates when the data in the list change. This allows `AddessBook` to only require one `Tag` object per unique tag, instead of each `Person` needing their own `Tag` objects.
```

Comment:
I think it should be 'when the data in the list changes'. Just minor grammar nits.